camera/Pilatus.py

```diff
@@ -231,23 +288,29 @@ class PilatusClass(PyTango.DeviceClass):
     class_property_list = {
         }
 
-
     #    Device Properties
     device_property_list = {
         'TmpfsSize' :
         [PyTango.DevInt,
          "Size of communication temp. filesystem (in MB)",0],
         }
 
-
     #    Command definitions
     cmd_list = {
         'getAttrStringValueList':
         [[PyTango.DevString, "Attribute name"],
          [PyTango.DevVarStringArray, "Authorized String value list"]],
+        'sendCamserverCmd':
+        [[PyTango.DevString, "Camserver command to send"],
+         [PyTango.DevVoid, "None"]],
+        'setReadoutGeometryCmd':
+        [[PyTango.DevString, "Set readout geometry"],
+         [PyTango.DevVoid, "None"]],
+        'getReadoutGeometryCmd':
```

Comment:
If you change getReadoutGeometryCmd setReadoutGeometryCmd to an attribute, then in spec we'll have enumeration introspection with getAttrStringValueList...